src/components/interfaces/HMI_API.xml

```diff
@@ -1879,10 +1879,10 @@
    <param name="hdRadioEnable" type="Boolean" mandatory="false">
      <description> True if the hd radio is on, false is the radio is off</description>
    </param>
-   <param name="availableHDs" type="Integer" minvalue="1" maxvalue="7" mandatory="false">
```

Comment:
availableHDs should be left in the HMI API until it is removed in the latest version of the Mobile API. Because the parameter is only deprecated, an integrator can still technically support the parameter in their hmi implementation.

Reply:
A <description> note should be left for this param describing that it is deprecated.

Reply:
@JackLivio Thanks. Fixed in be2dd5badfc9fef8734b56fce1a1827f8ac1aee0